src/js/hca/containers/ConfirmationPage.jsx

```diff
@@ -22,33 +22,49 @@ export class ConfirmationPage extends React.Component {
   }
 
   render() {
-    const time = _.get('form.submission.response.timestamp', this.props);
+    const { form } = this.props;
+    const { response } = form.submission;
+    const time = _.get('timestamp', response);
+    const name = form.data.veteranFullName;
+    const healthEligibilityCenter = [
```

Comment:
I think that @stacyhlinn had decided to not show any address here (since we don't get one back from the VA system).

Reply:
Thanks @jbalboni, @stacyhlinn @peggygannon could you please confirm that the address should be included? There is a [comment](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2965#issuecomment-330575036) that suggests that it should, also @peggygannon included this in her review comments.